core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -832,6 +820,24 @@ oppia.directive('conversationSkin', [
                     // There is a new card. If there is no feedback, move on
                     // immediately. Otherwise, give the learner a chance to read
                     // the feedback, and display a 'Continue' button.
+                    if (isFinalQuestion) {
```

Comment:
Didn't we discuss that this should "look" to the conversation skin service like an "add new card" operation? The question --> exploration transition should happen entirely within the exploration player state service.